arch/lkl/kernel/setup.c

```diff
@@ -41,14 +41,22 @@ static void __init lkl_run_kernel(void *arg)
 }
 
 int __init lkl_start_kernel(struct lkl_host_operations *ops,
-			unsigned long _mem_size,
+			const char *_mem_size,
 			const char *fmt, ...)
 {
 	va_list ap;
 	int ret;
 
 	lkl_ops = ops;
-	mem_size = _mem_size;
+	mem_size = memparse(_mem_size, NULL);
+	if (mem_size < 16 * 1024 * 1024) {
```

Comment:
thanks, this can be removed.

in fact less than 5M memsize makes my lkl apps out of memory during an execution so there is a limit around there.  but I'm okay to remove this check.